src/search.py

```diff
@@ -0,0 +1,33 @@
+from flask import Blueprint, request, render_template, session
+import require as require
+from require import response
+from flask_login import current_user
+from sql.auth import *
+from sql.inventory.getters import *
+
+search_blueprint = Blueprint("search",__name__,template_folder="../templates")
+
+@require.fields(request)
+def fetch_items(search_input):
+    if not search_input:
+        return render_template("index.html", user = current_user, items=[])
+    search_input = '%' + search_input + '%'
+    items_searched = get_item_by_search_name(search_input)
+    if items_searched is None:
+        items_searched = []
+    return render_template("index.html", user = current_user, items=items_searched)
+    
+
+@search_blueprint.route("/search", methods =["GET", "POST"])
+def search_database():
+    if request.method == "POST":
+        session["items"] = fetch_items()
```

Comment:
If the user searched for an empty string I think it should return you to the index page, since they want to view all products